app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -131,6 +131,16 @@ def textual_placement_groups
     h
   end
 
+  def textual_resource_pools
+    num   = @record.try(:resource_pools) ? @record.number_of(:resource_pools) : 0
+    h     = {:label => _('Resource Pools'), :icon => "pficon pficon-resource-pool", :value => num}
+    if num.positive? && role_allows?(:feature => "resource_pool_show_list")
+      h[:title] = _("Show all Resource Pools")
+      h[:link] = ems_cloud_path(@record.id, :display => 'resource_pools')
+    end
+    h
+  end
+
```

Comment:
This looks at least very similar to the `textual_resource_pools` method in the `ResourcePoolHelper` could we reuse that one?

Reply:
The h[:link] options in the `textual_resource_pools` methods of both files looks different. Perhaps we can't reuse it

Reply:
Link from `ems_cloud_helper` 
`/ems_cloud/5?display=resource_pools 
`
Link from `resource_pool_helper`
`/resource_pool/show/5?display=resource_pools`